microsetta_interface/server_config.json

```diff
@@ -1,6 +1,9 @@
 {
   "endpoint": "http://localhost:8083",
   "private_api_endpoint": "http://localhost:8082/api",
+  "public_api_endpoint": "http://localhost:8084/results-api",
+  "taxonomy_resource": "greengenes",
```

Comment:
Aren't the availalbe resources discoverable from the public api?

Reply:
Sure.
If we wanted to use that the options I see would be something like:
1) Choose one of the available taxonomy resources. E.g., just grab the first one in the list, or lexicographical minimum of them. Caveat: Does not guarantee we will use the specific resource we would like, if there is more than one. Seems a little weird.
2) Give the user a select picker to choose from. But then this has the caveats:
    a) they either see whatever variable names you gave the resources in the microsetta-public-api config
    b) OR we have to manually remap the variable names you gave them to things that appear to make more sense to the user
    c) This option doesn't really make sense if there is only one taxonomy resource corresponding to the given sample
3) You manually configure the taxonomy resource that the users see on their 16S samples, as above. Caveat: is not as flexible in the long run.

Reply:
Okay thanks. Let's continue fwd with (3) and revise as the needs of the page grow